src/components/ContentPreview/ContentPreview.js

```diff
@@ -113,7 +115,10 @@ class ContentPreview extends PureComponent<Props, State> {
         onLoad: noop,
         onMetric: noop,
         onNavigate: noop,
-        collection: []
+        collection: [],
+        contentSidebarProps: {
```

Comment:
PreviewDialog.js needs to change

Reply:
even if it only passes in `hasSidebar` currently? In the future when the explorer supports passing in props through the sidebar then it will have to forward them as well.

Reply:
nvm, I thought showSidebar was being passed from it